### ISSUE TITLE
Logback.xml in the configuration server JAR prevents custom log configurations

### ISSUE THREAD

**Issue by mariusoe** [inspectIT/inspectit-ocelot#639]
Currently, the configuration server's JAR contains a `logback.xml` file. This is quite handy, because the server is shipped including a nice log format.

Unfortunately, this prevents users from using their custom `logback.xml` file. Defining custom `logback.xml` file using the `loggin.config` or `logback.configurationFile` property is not working but the bundle one is used.

This has to be fixed in order to be able to use external log configurations.

**Comment by arolfes** [comment]
Hi @mariusoe ,
I'm not sure what you mean with `loggin.config` Parameter
Is it a `-D` JVM param or a param from config file?
In the PR it is implemented as `-D`
And I changed the name to `logging.config`
Regards Alex

**Comment by mariusoe** [comment]
Hi @arolfes 

this ticket is not about the agent's logging capability but about the configuration server's one (`components/inspectit-ocelot-configurationserver`).

The problem is - or at least I was not able to do this: I had to use a custom logging configuration in order to disable or change the logging output, but the configuration server ignored all attempts to do this and always used the default logging configuration file (`logback.xml`) which is bundled in the configuration server's JAR file.
Basically I wanted to achive something like descriped in the following stackoverflow post: https://stackoverflow.com/a/21725915/2478009
So I want to specify a custom logging configuration file which then is used by the server.

I'm not sure if this is already possible and I did something wrong or if there are some problems with the bundled `logback.xml`

**Comment by arolfes** [comment]
ok, 

I could reproduce it on my machine with the latest master

this command `java -Dlogback.configurationFile=logback-playground.xml -jar build/libs/inspectit-ocelot-configurationserver-SNAPSHOT.jar` should use the logback-playground.xml but it uses an default logback config (probably from classpath)

i will have a closer look on it

**Comment by arolfes** [comment]
So as I understood Spring boot always ignores the `-Dlogback.configurationFile` because it doesn't the property. You must use the `-Dlogging.config=logback-playground.xml`

I updated the README.md and then I would say this ticket can be closed.
